packages/helm-charts/testnet/templates/validators.statefulset.yaml

```diff
@@ -155,7 +155,7 @@ spec:
           [[ "$GETH_DEBUG" == "true" ]] && PROXY_ALLOW_PRIVATE_IP_FLAG="--proxy.allowprivateip"
```

Comment:
We could remove this and all references to `GETH_DEBUG` now

Reply:
The other option, which I believe @kevinjue tested yesterday, is to add `GETH_DEBUG=true` to .env.  I'm fine with getting rid of it, though.

Reply:
True! Thanks. Changed in https://github.com/celo-org/celo-monorepo/pull/2968